src/radical/entk/appman/appmanager.py

```diff
@@ -849,15 +727,14 @@ def pipeline_update(msg, reply_to, corr_id, mq_channel):
 
                     msg = json.loads(body)
 
-                    self._prof.prof('received obj with state %s for sync' %
-                                    msg['object']['state'], uid=msg['object']['uid'])
+                    self._prof.prof('received obj with state %s for sync' % msg['object']['state'], 
+                                    uid=msg['object']['uid'])
 
```

Comment:
The  first arg for `prof.prof` should be a short event name, like `obj_rcv`.  You can add a longer text in the `msg` field - but will you ever parse this msg in aalytics?  the profiler is not a logger ;-)

Reply:
Tbh I was doing that for some of the adaptation events :P But I shouldn't. Will change now.